documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -0,0 +1,368 @@
+2. After reviews: [https://balsamiq.cloud/scs7giw/ponj59g/r9840](https://balsamiq.cloud/scs7giw/ponj59g/r9840)
+
+#### Director
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/rD5AC](https://balsamiq.cloud/scs7giw/ponj59g/rD5AC)
+
+---
+
+## `/applications/[type]/[guid]?mode=assignment`
+
+### Application assignment mode
+
+The “home page” of a specific application during assignment of reviewers.
+
+The Supervisor -- with the proper permission to assign reviewers -- can do the assignment of reviewers to one (or more) section when the application is submitted to a new stage (the assignment changes per stage). After assigned the reviewers can review the specific sections.
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/r3DC3](https://balsamiq.cloud/scs7giw/ponj59g/r3DC3)
+
+---
+
+## `/applications/[type]/[guid]?mode=consolidation`
+
+### Application consolidation mode
+
+The “home page” of a specific application during consolidation of reviewers.
+
+The Supervisor or Chief user (Consolidator) can select the LOQ (List of questions) to be sent back to the Applicant. Edit the comments or the status of a reviewed question is allowed.
+
+On clicking the 'Save changes' button will save it for later and the 'Continue' button will load the Consolidation summary page.
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/r5273](https://balsamiq.cloud/scs7giw/ponj59g/r5273)
+
+---
+
+## `/applications/[type]/[guid]/summary?mode=consolidation`
+
+### Application consolidation mode summary page
+
+The summary page for a Consolidation.
+
+Shows the LOQ (List of questions) within the section, question number, status and comments. The user making the consolidation can select their decision and an overall comment.
+
+The preview of the message to be sent to the Applicant is shown based on final decision.
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/rD7FD](https://balsamiq.cloud/scs7giw/ponj59g/rD7FD)
+
+---
+
+## `/applications/[type]/[guid]/approval`
+
+### Application approval page
+
+The final page that only the Director sees where they can apply the final approval or rejection of an application.
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/rA5E5](https://balsamiq.cloud/scs7giw/ponj59g/rA5E5) (navigated to from Overview page)
+
+---
+
+## `/admin`
+
+### Admin home page
+
+Overview of Admin activity (New users, recently reviewed applications, etc.)
+
+---
+
+## `/admin/templates`
+
+### Application Template list
+
+Basically the same kind of thing as the Application list, but for templates
+
+---
+
+## `/admin/templates/new`
+
+### New Template launcher
+
+Initial template creation start page, where user enters basic unique info (Name, code, etc). Once this page is submitted, a new Template record is created in the database.
+
+Example: [https://balsamiq.cloud/scs7giw/ponj59g/r3C4F](https://balsamiq.cloud/scs7giw/ponj59g/r3C4F)
+
+---
+
+## `/admin/templates/[name]/[stage]/[section]/[page]`
```

Comment:
```suggestion
## `/admin/templates/[guid]/[section]/[page]`
```

I think the stage is not part of the URL anymore

Reply:
Yeah, there's probably a few other tweaks that need to be done like this -- will update this doc in the Routing PR to match what is actually implemented.